dashboard/test/controllers/api/v1/users_controller_test.rb

```diff
@@ -87,6 +87,35 @@ class Api::V1::UsersControllerTest < ActionController::TestCase
     refute @user.show_progress_table_v2
   end
 
+  test 'a post request to disable_lti_roster_sync updates lti_roster_sync_enabled' do
+    teacher = create :teacher
+    teacher.lti_roster_sync_enabled = true
```

Comment:
Tiny nitpick - I think you can just do `teacher = create :teacher, lti_roster_sync_enabled: true` to make it a one-liner.